packages/react-web/src/inbox/KeyBundleStorage.ts

```diff
@@ -0,0 +1,19 @@
+import { EthereumAddress } from '@lens-protocol/shared-kernel';
+import { BaseStorageSchema, IStorageProvider, Storage } from '@lens-protocol/storage';
+import { z } from 'zod';
+
+export const KeyBundleData = z.string();
+
+export type KeyBundleData = z.infer<typeof KeyBundleData>;
+
+export function createKeyBundleStorage(
+  storageProvider: IStorageProvider,
+  namespace: string,
+  address: EthereumAddress,
+) {
+  const notificationStorageDataSchema = new BaseStorageSchema(
+    `lens.${namespace}.${address}.inbox.keyBundle`,
```

Comment:
ah... interesting the use of `address`, I can see a narrative with this, wondering if you could spend 5 minutes to explain to me the rationale you had in mind (also in relation to the comment I left on the use case interactors).

Reply:
I copied this approach from how xmtp stores keys, as a key bundle is created with an address (signer), then I think it is important to store a key per address, if a user changes wallet, then we can retrieve the correct key from the storage for the current signer, the same with saving, to avoid overwriting the previous key.

Reply:
I am not sure if we should leave XMTP credentials laying around after login. I think this ties up with the idea of clearing the connection from `WalletLogout` and this https://github.com/lens-protocol/lens-sdk/pull/361#discussion_r1222654977 convesation.